charts/render.py

```diff
@@ -212,17 +213,20 @@ def main():
             )
         )
 
-    sys.stdout.write(
-        START_BLOCK.format(
-            configlines='\n'.join(
-                ("{{%- set {} = salt.metalk8s_service_configuration"
-                 ".get_service_conf('{}', '{}') %}}").format(
-                    service_config[0], args.namespace, service_config[1]
-                ) for service_config in args.service_configs
-            )
-        ).lstrip()
-    )
-    sys.stdout.write('\n')
+    if args.service_configs:
+        sys.stdout.write(
+            START_BLOCK.format(
+                configlines='\n'.join(
+                    ("{{%- set {} = salt.metalk8s_service_configuration"
+                     ".get_service_conf('{}', '{}') %}}").format(
+                        service_config[0], args.namespace, service_config[1]
+                    ) for service_config in args.service_configs
+                )
+            ).lstrip()
+        )
+        sys.stdout.write('\n')
+    else:
+        sys.stdout.write(START_BLOCK.format(configlines='').lstrip())
```

Comment:
Would prefer an `if` only to set a `configlines` variable and then having the write with format